packages/instant-apps-components/src/instant-apps-landing-page-2.html

```diff
@@ -0,0 +1,132 @@
+    >
+    </instant-apps-landing-page>
+    <calcite-shell class="calcite-mode-light">
+      <instant-apps-header
+        id="instant-apps-header"
+        slot="header"
+        title-text="Instant Apps: Header (Shared Theme)"
+        text-color="#fff"
+        background-color="#447f0c"
+        logo-image="https://www.esri.com/content/dam/esrisites/en-us/common/icons/product-logos/arcgis-instant-apps-64.svg"
+        logo-image-alt-text="ArcGIS Instant Apps logo"
+        logo-link="https://www.esri.com/en-us/arcgis/products/arcgis-instant-apps/overview"
+        info-button="true"
+        info-title-text="Open information"
+        info-is-open="true"
+      >
+        <instant-apps-social-share slot="actions-end" share-button-color="inverse" popover-button-icon-scale="s"></instant-apps-social-share>
+      </instant-apps-header>
+      <calcite-shell-panel slot="panel-start">
+        <calcite-action-bar slot="action-bar">
+          <calcite-action-group slot="bottom-actions">
+            <calcite-action id="back" icon="content-minimal" text="Landing page"></calcite-action>
+          </calcite-action-group>
+        </calcite-action-bar>
+        <calcite-panel>
+          <div id="legend"></div>
+        </calcite-panel>
+      </calcite-shell-panel>
+      <div id="viewDiv"></div>
+    </calcite-shell>
+    <script>
+      require(['esri/WebMap', 'esri/views/MapView', 'esri/widgets/Legend', 'esri/core/reactiveUtils', 'esri/config'], (WebMap, MapView, Legend, reactiveUtils, esriConfig) => {
+        esriConfig.portalUrl = 'https://prodtesting.maps.arcgis.com/';
+
+        const map = new WebMap({
+          portalItem: {
+            id: 'a235bfc2d24b4568a3b2832b3336d0cc',
+          },
+        });
+
+        const view = new MapView({
+          map,
+          container: 'viewDiv',
+        });
+
+        new Legend({
+          view,
+          container: 'legend',
+        });
+
+        const landingPage = document.querySelector('instant-apps-landing-page');
+        view.when(() => {
+          landingPage.portal = view.map.portalItem.portal;
+          landingPage.oauthappid = 'wfblKnj2mvxLOiCx';
+        });
+      });
+    </script>
+    <script>
+      const backButton = document.getElementById('back');
+      const landingPage = document.querySelector('instant-apps-landing-page');
+      backButton.addEventListener('click', () => {
+        landingPage.open = true;
+      });
+      document.addEventListener('landingPageOpen', e => {
+        console.log('Landing page is open.', e);
```

Comment:
Are these temporary for debugging?

Reply:
This is just the html file used to test it, won't show up in any app. I copied the original and just added the sign in capability.